### PR TITLE
Add different card color based on levels

### LINKED ISSUE #161: Different card color based on levels
Life could be a dream 🎶 
Sh-boom, if anyone could change colors for levels 🎶 

image

### PR DESCRIPTION
**Pull request by Nandika-A**
resolves #161 

A color is linked with each level by declaring the tuple LEVEL_COLORS in project-models/issues. This tuple is given as a choice at the field "levelcolor".  The values in this field are filled automatically using project-views/populate_issues, by assigning the level number to the levelcolor field.
The following colors are linked with the levels:
1. FREE : #f2f2f2
2.  EASY : #00b74a
3.  MEDIUM : #ffa900
4.  HARD : #f93154
5.  VERY_EASY : default

### CODE REVIEW

File: home/templates/dashboard/index.html [hunk 1]
```diff
@@ -113,7 +113,7 @@ <h6 class="">
 
                             <p class="card-text">
                                 <a class="btn btn-rounded btn-info" data-mdb-toggle="tooltip" title="Mentor">@{{ issue.mentor }}</a>
-                                <a class="btn btn-rounded btn-info" data-mdb-toggle="tooltip" title="Level">{{ issue.get_level_display }}</a>
+                                <a class="btn btn-rounded btn-info" data-mdb-toggle="tooltip" title="Level" style="background-color: {{ issue.get_levelcolor_display }} !important;">{{ issue.get_level_display }}</a>
```
Comment (PriyavKaneria) [inline home/templates/dashboard/index.html:116]: Is it absolutely necessary to use important here? If can be avoided, please do. 
Inline styles should generally override other styles

File: project/models.py [hunk 2]
```diff
@@ -41,6 +41,8 @@ class Issue(models.Model):
     FREE, EASY, MEDIUM, HARD, VERY_EASY = 0, 1, 2, 3, 4
     FREE_READ, VERY_EASY_READ, EASY_READ, MEDIUM_READ, HARD_READ = "Free", "Very-Easy", "Easy", "Medium", "Hard" \
         # Human Readable Names
+    FREE_COLOR, VERY_EASY_COLOR, EASY_COLOR, MEDIUM_COLOR, HARD_COLOR = "#f2f2f2", "", "#00b74a", "#ffa900", "#f93154" \
```
Comment (PriyavKaneria) [inline project/models.py:44]: Very_easy_color seems to be empty, even if it is set by default add the color code

File: project/migrations/__init__.py
Comment (PriyavKaneria) [inline project/migrations/__init__.py:1]: Migrations are important if someone does forget to migrate manually, reset the file deletions for the migrations folder

**Review by PriyavKaneria — CHANGES_REQUESTED** [review]
Please check out the mentioned changes

File: home/templates/dashboard/index.html (hunk 1 above)
Comment (Nandika-A) [reply home/templates/dashboard/index.html:116]: Without important, the default color is being considered.

**Comment by Nandika-A** [comment]
@PriyavKaneria I have reset the migrations folder deletions and provided the default color code. Please check if any changes are required.

**Review by PriyavKaneria — APPROVED** [review]
Lgtm

**Comment by PriyavKaneria** [comment]
Build is failing due to flake8 linting
Please resolve the minor lint issue so I can merge

**Comment by Nandika-A** [comment]
> @Nandika-A Build is failing due to flake8 linting Please resolve the minor lint issue so I can merge

Sir this lint failure is because one line has more than 119 characters in project-models/issues. When I break it into 2 lines, it shows invalid syntax in python. 
`FREE_COLOR, VERY_EASY_COLOR, EASY_COLOR, MEDIUM_COLOR, HARD_COLOR = "#f2f2f2", "#39c0ed", "#00b74a", "#ffa900", "#f93154" \`
Should I assign these values individually?

**Comment by Nandika-A** [comment]
@PriyavKaneria I have now put these level colors in different lines.

**Review by PriyavKaneria — APPROVED** [review]
Works

**Pull request merged**